### PR TITLE
Add Another Pine Forest Mod

### PR DESCRIPTION
**Pull request by wxMichael**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -6867,3 +6867,27 @@ plugins:
...
+
+  - name: 'Another Pine Forest (Mod|Trees Only|Winter)\.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/54027/' ]
+    inc: [ 'Boston Natural Surroundings.esp' ]
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Another Pine Forest plugin' ]
+        condition: 'many("Another Pine Forest (Mod|Trees Only|Winter)\.esp")'
+    dirty:
+      # Main v0.3
```
Comment (sibir-ine) [inline masterlist.yaml]: You don't need to (& it doesn't make sense to) put cleaning data in a regex plugin when it only applies to a specific plugin. Also, version comments aren't necessary as those will be recorded in the commit message:

```suggestion
  - name: 'Another Pine Forest Mod.esp'
    dirty:
```

File: masterlist.yaml [hunk 2]
```diff
@@ -6867,3 +6867,27 @@ plugins:
...
+      - <<: *quickClean
+        crc: 0x6ABC1475
+        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+        itm: 2
+      # Trees Only v0.3a
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
  - name: 'Another Pine Forest Trees Only.esp'
    dirty:
```

File: masterlist.yaml [hunk 3]
```diff
@@ -6867,3 +6867,27 @@ plugins:
...
+      - <<: *quickClean
+        crc: 0xC90068E2
+        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+        itm: 2
+      # Winter v0.3b
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
  - name: 'Another Pine Forest Winter.esp'
    dirty:
```

File: masterlist.yaml [hunk 4]
```diff
@@ -6867,3 +6867,27 @@ plugins:
     clean:
       - crc: 0x17ED1071
         util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+
+  - name: 'Another Pine Forest (Mod|Trees Only|Winter)\.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/54027/' ]
```
Comment (sibir-ine) [inline masterlist.yaml]: LOOT's UI makes better use of location names now, so it helps to add them:

```suggestion
    url:
      - link: 'https://www.nexusmods.com/fallout4/mods/54027/'
        name: 'Another Pine Forest Mod'
```

**Review by sibir-ine — DISMISSED** [review]
@wxMichael, looks good, just needs a few changes.

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
Thanks.

**Pull request merged**